man/cc-test-reporter-format-coverage.1.md

```diff
@@ -33,8 +33,7 @@ tests were run. Defaults to current working directory.
 
 ## COVERAGE_FILE
 
-Path where the coverage file is located. It requires the flag `--input-type`
-to be set.
+(Optional) Explicit path where the coverage file is located. If not passed, the path to the file is inferred based on standard some coverage tool locations. If specifying an explicit COVERAGE_FILE, an `--input-type` must also be specified.
```

Comment:
@noahd1 i agree on 
>  I think we want to change the behavior to an error message for now.
i don't know if we needed for `--input-type` someone could run 
```
cc-test-reporter format-coverage --input-type lcov -o coverage.lcov.json
cc-test-reporter format-coverage --input-type simplecov -o coverage.simplecov.json
... # sum and upload
```
^ i think this is valid usage, and it will search for the harcoded lcov file path and the simplecov file